specs/core/0_beacon-chain.md

```diff
@@ -311,27 +321,101 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 }
 ```
 
-#### `ProposalSignedData`
+#### Deposits
+
+##### `Deposit`
 
 ```python
 {
-    # Slot number
+    # Receipt Merkle branch
+    'merkle_branch': '[hash32]',
+    # Merkle tree index
+    'merkle_tree_index': 'uint64',
+    # Deposit data
+    'deposit_data': {
+        # Deposit parameters
+        'deposit_parameters': DepositParameters,
+        # Value in Gwei
+        'value': 'uint64',
+        # Timestamp from deposit contract
+        'timestamp': 'uint64',
+    },
+}
+```
+
+##### `DepositParameters`
```

Comment:
Maybe `DepositInput`?

Reply:
This mirrors the input arg in the pow contract
```python
def deposit(deposit_parameters: bytes[2048])
```

Going to leave as is. We can change both in a separate PR if you want